Refresh.GameServer/Types/Commands/Command.cs

```diff
@@ -0,0 +1,3 @@
+namespace Refresh.GameServer.Types.Commands;
+
+public record Command(string Name, string? Arguments);
```

Comment:
Might be possible to use a ref struct here, we dont store commands persistently.

Also, the name could be improved. Something like CommandData? CommandInvocation?